packages/core/test/hotkeys/hotkeysTests.tsx

```diff
@@ -362,7 +361,7 @@ describe("Hotkeys", () => {
             verifyCombos(
                 Array.apply(null, Array(26)).map((_: any, i: number) => {
                     const combo = String.fromCharCode(alpha + i).toUpperCase();
-                    const event = { which: alpha + i } as KeyboardEvent;
+                    const event: KeyboardEvent = { which: alpha + i } as any;
```

Comment:
Same.

Reply:
being truthful about escaping type safety

Reply:
also it doesn't compile without the `as any`. that's the motivation for the lint rule -- stricter checks